source/_reference-a-f.html.md.erb

```diff
@@ -5024,13 +5024,13 @@ Lists all cross-sheet references for the sheet.
 
 # Dashboards
 
-Smartsheet Dashboards are a collection of widgets that can contain data from a variety of different data sources (for example, sheets, reports, or custom data). Dashboards were once called Sights(TM) and all endpoints still use "sights" in the path. 
+Smartsheet Dashboards are a collection of widgets that can contain data from a variety of different data sources (for example, sheets, reports, or custom data). Dashboards were once called Sights(TM) and this name is still present in object names, endpoint paths, and other places. 
 
 ## <%= OBJECTS %>
 
-### Dashboard Object aka Sight Object
+### Sight Object
```

Comment:
I'm not sure if we want to be changing the existing places we used "Dashboard" to "Sight". This is still a "Dashboard object", and still uses a "Dashboard Id"